src/components/NewLexemeForm.vue

```diff
@@ -51,6 +51,9 @@ const lexicalCategory = computed( {
 	},
 	set( newLexicalCategory: SearchedItemOption | null ): void {
 		store.commit( SET_LEXICAL_CATEGORY, newLexicalCategory );
+		if ( newLexicalCategory ) {
+			store.commit( CLEAR_PER_FIELD_ERRORS, 'lexicalCategoryErrors' );
```

Comment:
This effectively hard-codes the knowledge that “no lexical category selected” is the only possible error for this field. That feels a bit awkward to me, but I can’t think of a great way to avoid it.

Reply:
yeah, I think after all these per-field-errors are done, it might be sensible to have another overall look and think about whether we can come up with a better solution. Though the result might be not to change it, because yagni as we don't have any other errors yet, might never have any other errors and if we have other errors whereever they might be coming from might determine the best way to set this up.

(though it will be interesting to look at the "lemma too long error")